### PR TITLE
4.6.0 upgrade path is to pass 4.6.1 to create the extra view in there

### PR DESCRIPTION
**Pull request by DaanHoogland**
test build done only

### CODE REVIEW

**Comment by bhaisaab** [comment]
LGTM, thanks @DaanHoogland for fixing this so fast

**Comment by bhaisaab** [comment]
once merged on master, I'll rebase quota-master branch (the quota PR) with latest master and then fix it's db upgrade paths.

File: setup/db/db/schema-460to470.sql [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
--- KIND, either express or implied.  See the License for the
--- specific language governing permissions and limitations
--- under the License.
-
---;
--- Schema upgrade from 4.6.0 to 4.7.0;
---;
-
-CREATE TABLE IF NOT EXISTS `cloud`.`domain_vlan_map` (
```
Comment (pdube) [inline setup/db/db/schema-460to470.sql]: Is this in the 4.6.0 to 4.6.1?

Comment (ustcweizhou) [inline setup/db/db/schema-460to470.sql]: yes

Comment (DaanHoogland) [inline setup/db/db/schema-460to470.sql]: actually it is in schema-461to470

**Comment by pdube** [comment]
LGTM by code review.

File: setup/db/db/schema-460to470.sql (hunk 1 above)
Comment (ustcweizhou) [inline setup/db/db/schema-460to470.sql]: @DaanHoogland I added it to both 460to470 and 461to470

Comment (DaanHoogland) [inline setup/db/db/schema-460to470.sql]: @ustcweizhou 460to470 is to be removed. all upgrades wil go 4.6.0 -> 4.6.1 -> 4.7.0

**Comment by bhaisaab** [comment]
Merging now, that we've enough LGTMs.

**Pull request merged**